modules/cfe_testcase/src/evs_filters_test.c

```diff
@@ -0,0 +1,49 @@
+/*************************************************************************
+**
+**      GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**      Copyright (c) 2006-2019 United States Government as represented by
+**      the Administrator of the National Aeronautics and Space Administration.
+**      All Rights Reserved.
+**
+**      Licensed under the Apache License, Version 2.0 (the "License");
+**      you may not use this file except in compliance with the License.
+**      You may obtain a copy of the License at
+**
+**        http://www.apache.org/licenses/LICENSE-2.0
+**
+**      Unless required by applicable law or agreed to in writing, software
+**      distributed under the License is distributed on an "AS IS" BASIS,
+**      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**      See the License for the specific language governing permissions and
+**      limitations under the License.
+**
+** File: evs_filters_test.c
+**
+** Purpose:
+**   Functional test of basic EVS Reset Filters APIs
+**
+**   Demonstration of how to register and use the UT assert functions.
+**
+*************************************************************************/
+
+/*
+ * Includes
+ */
+
+#include "cfe_test.h"
+
+void TestResetFilters(void)
+{
+    UtPrintf("Testing: CFE_EVS_ResetFilter, CFE_EVS_ResetAllFilters");
+
+    UtAssert_INT32_EQ(CFE_EVS_ResetFilter(1), CFE_SUCCESS);
```

Comment:
Suggest a comment here, something like - `Test logic below depends on the test case registering an EID of 1 and not registering 0, and the resets in theory could impact test behavior/management related to events.  Although the expectation is either all or none of an EID would be filtered (no use case for a "counting" filter within the test app) so for nominally this is no impact.`  Provides clues for maintenance.